lib/view_model/dashboard/transaction_list_item.dart

```diff
@@ -57,33 +57,48 @@ class TransactionListItem extends ActionListItem with Keyable {
     if (transaction.direction == TransactionDirection.incoming) {
       if (balanceViewModel.wallet.type == WalletType.monero ||
           balanceViewModel.wallet.type == WalletType.haven) {
-          return formattedPendingStatus;
-        }
+        return formattedPendingStatus;
       }
-    return transaction.isPending ? S.current.pending : '';
     }
+    return transaction.isPending ? S.current.pending : '';
+  }
 
   String get formattedFiatAmount {
     var amount = '';
 
-    switch(balanceViewModel.wallet.type) {
+    switch (balanceViewModel.wallet.type) {
       case WalletType.monero:
         amount = calculateFiatAmountRaw(
-          cryptoAmount: monero!.formatterMoneroAmountToDouble(amount: transaction.amount),
-          price: price);
+            cryptoAmount: monero!.formatterMoneroAmountToDouble(amount: transaction.amount),
+            price: price);
         break;
       case WalletType.bitcoin:
       case WalletType.litecoin:
         amount = calculateFiatAmountRaw(
-          cryptoAmount: bitcoin!.formatterBitcoinAmountToDouble(amount: transaction.amount),
-          price: price);
+            cryptoAmount: bitcoin!.formatterBitcoinAmountToDouble(amount: transaction.amount),
+            price: price);
         break;
       case WalletType.haven:
         final asset = haven!.assetOfTransaction(transaction);
         final price = balanceViewModel.fiatConvertationStore.prices[asset];
         amount = calculateFiatAmountRaw(
-          cryptoAmount: haven!.formatterMoneroAmountToDouble(amount: transaction.amount),
-          price: price);
+            cryptoAmount: haven!.formatterMoneroAmountToDouble(amount: transaction.amount),
+            price: price);
+        break;
+      case WalletType.ethereum:
+        final asset = ethereum!.assetOfTransaction(balanceViewModel.wallet, transaction);
+        final price = balanceViewModel.fiatConvertationStore.prices[asset];
+        amount = calculateFiatAmountRaw(
+            cryptoAmount: ethereum!.formatterEthereumAmountToDouble(transaction: transaction),
+            price: price);
+        break;
+      case WalletType.nano:
+        final nanoTransaction = transaction as NanoTransactionInfo;
+        amount = calculateFiatAmountRaw(
+            cryptoAmount:
+                NanoUtil.getRawAsDecimal(nanoTransaction.amountRaw.toString(), NanoUtil.rawPerNano)
+                    .toDouble(),
+            price: price);
         break;
```

Comment:
we cannot use any subpackage in the main code (`cw_ethereum`, `cw_nano`, etc...) as they are not defined to the project when building Monero.com, we instead use `ethereum/ethereum.dart` where the `ethereum!.` is coming from, same goes for the Nano Util, you must find another way to use it here if necessary, i.e through the nano instance that should be defined in `nano/nano.dart`